### PR TITLE
CC-1974: NFSClient disabled does not break serviced on a remote agent

### PR DESCRIPTION
**Pull request by smousa**
TODO: rename NFSDriver to something that doesn't say nfs
* Added an option on the storage driver to disable networking
* Use the option for when the nfs client is disabled
* Make the varpath configurable for isvcs and volumes on the agent

### CODE REVIEW

**Comment by wwwtyro** [comment]
LGTM

File: volume/nfs/nfs.go [hunk 1]
```diff
@@ -98,11 +100,12 @@ func (d *NFSDriver) Get(volumeName string) (volume.Volume, error) {
 		driver: d,
 		tenant: getTenant(volumeName),
 	}
-	//actual NFS mount
-	if err := mount(volumeName, volumePath); err != nil {
-		return nil, err
+	if !d.networkDisabled {
```
Comment (jplouis) [inline volume/nfs/nfs.go]: Mounting will create the directory if it isn't there and then mount the remote FS.  What  creates the directory on the local filesystem if the network is disabled?

Comment (smousa) [inline volume/nfs/nfs.go]: setupVolume in node/agent.go

Comment (jplouis) [inline volume/nfs/nfs.go]: Thanks!

**Pull request merged**